### PR TITLE
Remove functions module

### PR DESCRIPTION
**Pull request by branchvincent**
## Why?
* Trying to remove outdated code (via a minimal change set)

## What changed?
- [x] Replace CLI's reliance on `functions` by replacing calls to `service` classes
- [x] Split normalized metric stuff (which was the only code not cleanly mapped to a single api call) from `functions.ems` to separate file `utils.normalization`
- [x] Removed `functions` module

## TODO
 Within `utils.normalization`, there is very specific/convoluted/broken logic. My opinion is if that file serves a general + valuable use-case, then a separate PR should fix it. Otherwise, it should be completely removed from this project and live elsewhere.

### CODE REVIEW

**Comment by bretcj7** [comment]
Any breaking changes with this PR?

**Comment by branchvincent** [comment]
Besides the removal of `functions`, there are no breaking changes. Only caveat is some of the cli commands may look slightly different while others are still be broken.

File: setup.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ def run(self):
         "dash-core-components",
         "dash-html-components",
         "dash",
+        "dataclasses",  # needed for python < 3.7
```
Comment (borregoho) [inline setup.py]: Can we make python 3.7+ a hard dependency?

Comment (branchvincent) [reply setup.py]: Good question, does not matter to me. I guess that should be answered by our intended audience, same as things like officially supported Windows, etc.

**Pull request merged**